dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNStaticVarCompensatorInterfaceIIDMNoStandByAutomaton.cpp

```diff
@@ -22,11 +22,13 @@
 
 #include "DYNStaticVarCompensatorInterfaceIIDMNoStandByAutomaton.h"
 
-extern "C" DYN::StaticVarCompensatorInterfaceIIDMExtensionNoStandByAutomaton* createExtension(powsybl::iidm::StaticVarCompensator& svc) {
+extern "C" DYN::StaticVarCompensatorInterfaceIIDMExtensionNoStandByAutomaton*
+createExtensionStaticVarCompensatorInterfaceIIDMExtension(powsybl::iidm::StaticVarCompensator& svc) {
```

Comment:
createStaticVarCompensatorInterfaceIIDMExtension would be better to avoid having Extension twice in the name

Reply:
OK